src/com/rjmetrics/sweet_liberty/core.clj

```diff
@@ -69,14 +70,16 @@
   (assoc-in sweet-lib-config [:liberator-config :handle-exception]
             (fn [ctx]
               (let [exception-info (:exception ctx)]
-                (if (-> exception-info ex-data :is-sweet-lib-exception?)
-                  (do
-                    (log-exception exception-info)
-                    (ring-response {:body (get-error-json exception-info
-                                                          (get-in sweet-lib-config [:options :return-exceptions?]))
-                                    :status 400
-                                    :headers {"Content-Type" "application/json; charset=UTF-8"}}))
-                  (handle-exception-fn ctx))))))
+                (if-not (-> exception-info ex-data :is-sweet-lib-exception?)
```

Comment:
Should we limit this behavior to exceptions marked w/ `:is-sweet-lib-exception?`, or just look for anything w/ `:ring-response` ?